releasenotes/notes/update-registry-non-atomic-c204f35fc48f9b82.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Update registry writer to not write atomically when Agent runs on ECS Fargate.
```

Comment:
Should we explain why this is an `enhancements` ? I'm not sure the current version of this releasenote is useful to users.